### PR TITLE
feat: auto release Python SDK MacOS in CICD

### PR DESCRIPTION
**Pull request by HuilinWu2**
linked [issue#1351](https://github.com/4paradigm/OpenMLDB/issues/1351)

### CODE REVIEW

File: steps/zoo.cfg [hunk 1]
```diff
@@ -9,7 +9,7 @@ syncLimit=5
 # the directory where the snapshot is stored.
 # do not use /tmp for storage, /tmp here is just 
 # example sakes.
-dataDir=/rambuild/ut_zookeeper
+dataDir=./data
```
Comment (aceforeverd) [inline steps/zoo.cfg:12]: no unrelated changes

File: .github/workflows/sdk.yml [hunk 2]
```diff
@@ -268,6 +268,61 @@ jobs:
         env:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+  
+  python-sdk-mac:
```
Comment (aceforeverd) [inline .github/workflows/sdk.yml:272]: checkout matrix : https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

File: .github/workflows/sdk.yml [hunk 3]
```diff
@@ -268,6 +268,61 @@ jobs:
...
+
+      - name: test sqlalchemy
+        run: |
+          bash steps/test_python.sh
+
+      - name: upload python ut results
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: mac-ut-result-python-${{ github.sha }}
```
Comment (aceforeverd) [inline .github/workflows/sdk.yml:313]: checkout `publish-test-results` job below, you might update `files` options there

**Review by aceforeverd — CHANGES_REQUESTED** [review]

File: steps/zoo.cfg (hunk 1 above)
Comment (HuilinWu2) [reply steps/zoo.cfg:12]: This change is due to the macos root permission, which caused the Error: `mkdir: /rambuild/ut_zookeeper: Read-only file system`. Therefore, we change the absolute path to relative path, which works for both Linux and Mac.

File: .github/workflows/sdk.yml (hunk 3 above)
Comment (tobegit3hub) [reply .github/workflows/sdk.yml:313]: Change for two options of public result job.

File: .github/workflows/sdk.yml (hunk 2 above)
Comment (HuilinWu2) [reply .github/workflows/sdk.yml:272]: may further improve it later due to the complexity of current process

File: .github/workflows/sdk.yml (hunk 3 above)
Comment (HuilinWu2) [reply .github/workflows/sdk.yml:313]: updated.

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by aceforeverd — APPROVED** [review]
